shavar/templates/swagger.mako

```diff
@@ -0,0 +1,100 @@
+swagger: "2.0"
+host: ${HOST}
+info:
+  title: Shavar Service
+  description: Mozilla's implementation of the safe browsing protocol
+  contact:
+    name: Ryan Tilder
+    email: rtilder@mozilla.com
+  license:
+    name: MPL v2.0
+    url: https://www.mozilla.org/en-US/MPL/2.0/
+  version: ${VERSION}
+  x-mozilla-services:
+    homepage: https://github.com/mozilla-services/shavar
+schemes:
+    - ${SCHEME}
+paths:
+    /__heartbeat__:
+      get:
+        operationId: getHeartbeat
+        description: Performs an heartbeat on the app
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: The hearbeat was successfull
+    /__version__:
+      get:
+        operationId: getVersion
+        description: Returns the version
+        produces:
+        - application/json
+        responses:
+          '200':
+            description: The version is returned
+    /list:
+      get:
```

Comment:
If the implementation supports both, we may as well document that.

I would however suggest that we make `GET` description the one that refers to the other one to avoid suggesting that it's the preferred method.